lib/index.js

```diff
@@ -39,8 +45,30 @@ module.exports = function(contents) {
     }
   })
 
-  return genCode.generate(res)
-    + (hasComponent ? genHotReload(id) : '')
+  if (hasComponent) {
```

Comment:
I prefer to negate this condition and `return` in this block early so that we can reduce the nesting level 🙂